libs/pageserver_api/src/key.rs

```diff
@@ -236,6 +236,14 @@ impl Key {
         field5: u8::MAX,
         field6: u32::MAX,
     };
+    pub const NON_L0_MAX: Key = Key {
```

Comment:
```suggestion
    /// A key slightly smaller than [`Key::MAX`] for use in layer key ranges to avoid them to be confused with L0 layers
    pub const NON_L0_MAX: Key = Key {
```